tools/travis_before_install.sh

```diff
@@ -28,33 +28,49 @@ retry () {
 }
 
 
-# on Python 2.7, use the system versions of numpy, scipy, and matplotlib
-# and the minimum version of cython and networkx
-if [[ $TRAVIS_PYTHON_VERSION == 2.7* ]]; then
+# on Python 2.7, use the system versions of matplotlib, numpy and scipy
+# which is also our minimum requirements
+# this will change if travis updates to ubuntu LTS 14.04
+# http://docs.travis-ci.com/user/ci-environment/#CI-environment-OS
+if [ $TRAVIS_PYTHON_VERSION == 2.7 ]; then
+    sudo apt-get install python-scipy python-matplotlib python-numpy python-imaging
+    # above should be equal to minimum requirements
+    # remove from requirements and set the rest to minimum version
+    sed -i '/^matplotlib.*$/d' requirements.txt
+    sed -i '/^scipy.*$/d' requirements.txt
+    sed -i '/^numpy.*$/d' requirements.txt
+    sed -i '/^pillow.*$/d' requirements.txt
+    sed -i 's/>=/==/g' requirements.txt
+    # remove pillow from wheel install
+    export WHEELBINARIES=${WHEELBINARIES/pillow/}
+    # use system python
     virtualenv --system-site-packages ~/venv
```

Comment:
Adding: I think all the 2.7 trouble are due to travis venv [has python 2.7.9](https://travis-ci.org/scikit-image/scikit-image/jobs/63883427#L98), the virtualenv package in 2.7 seem to be installed by system python and [we get python 2.7.3 in our venv](https://travis-ci.org/scikit-image/scikit-image/jobs/63883427#L1431). Which is fair enough, as we are going to check system installed packages (scipy, matplotlib, etc).

A work around would of course be to use the wheels, with the drawbacks:
1. Testing deps with wheels, not guaranteeing scikit-image will work with minimum requirements from system packages. This might not be as bad as we think, as operating systems do ship pre-releases and report them as stable packages (ref llvm 3.5 on Ubuntu 14.04). A pre release should not be considered the same as the pypi stable release, and on can potentially stumble upon API-changes that break the build.
2. We will need to compile PIL 1.1.7 to wheel, compile it from source every time or drop the PIL dep.
3. Might be more that I'm unaware of...